### PR TITLE
Collective voting components refactor

### PR DESCRIPTION
**Pull request by kwingram25**
- TxButton to functional
- Reuse components for council/tech committee

### CODE REVIEW

File: packages/react-components/src/Collective/Slashing.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+import { useTranslation } from '../translate';
+import useAvailableSlashes from './useAvailableSlashes';
+
+interface Option {
+  text: string;
+  value: number;
+}
+
+export default function Slashing ({ className, isMember, members = [] }: Props): React.ReactElement<Props> {
```
Comment (jacogr) [inline packages/react-components/src/Collective/Slashing.tsx]: This is def not collective - it is specific to council.

File: packages/react-components/src/Collective/Proposals.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <div className={className}>
+      <Button.Group>
+        <Propose
+          collective={collective}
+          isMember={isMember}
+          members={members}
+          prompt={proposePrompt}
+          thresholdHelp={thresholdHelp}
+        />
+        {isCouncil && (
```
Comment (jacogr) [inline packages/react-components/src/Collective/Proposals.tsx]: We really don't want this in here - we want the logic for council and the way it displays, in council. As soon as we start treating both the same, we are going to run into a mess here.

**Comment by jacogr** [comment]
Fo some reason the proposals are now disabled for me.

image

**Comment by jacogr** [comment]
Also cannot vote on the members. Existing vs new -

image

image

**Comment by kwingram25** [comment]
Hmm looks related to isMember check, I've always had it on since I only tried on testnet...

**Comment by jacogr** [comment]
Closing. We only have 2 `extends TxModal` left -

- claims app (not sure why this is a modal)
- inside contracts

So all the areas here are covered.

**Pull request closed without merging**